controllers/yawol-controller/loadbalancerset/loadbalancerset_controller.go

```diff
@@ -75,20 +76,25 @@ func (r *LoadBalancerSetReconciler) Reconcile(ctx context.Context, req ctrl.Requ
 	var (
 		readyMachineCount   int
 		deletedMachineCount int
+		setContainsMaster   bool
 	)
 
 	for i := range childMachines.Items {
 		if childMachines.Items[i].DeletionTimestamp != nil {
 			deletedMachineCount++
 			continue
 		}
+		if isMachineMaster(childMachines.Items[i]) {
```

Comment:
Would like to rename this to `isMachineKeepalivedMaster` to make it more clear. Same with the `setContainsMaster` var.